src/index.js

```diff
@@ -0,0 +1,143 @@
+const cityNameInput = document.getElementById('cityNameInput');
+const cityNameResetBtn = document.getElementById('cityNameReset');
+const headerCityName = document.getElementById('headerCityName');
+const tempValue = document.getElementById('tempValue');
+const gardenEmoji = document.getElementById('gardenContent');
+const currentTempButton = document.getElementById('currentTempButton');
+const applyCityArrow = document.getElementById('applyCity');
+const skySelection = document.getElementById('skySelect');
+const skyDisplay = document.getElementById('skyDisplay');
+let isFahrenheit = true;
+let currentTemp = 50;
+
+function updateCityName() {
+  const city = cityNameInput.value.trim();
+  if (city) {
+    headerCityName.textContent = city;
+  } else {
+    headerCityName.textContent = '';
+  }
+}
+
+document.getElementById('toggleTempUnit').addEventListener('click', () => {
+  isFahrenheit = !isFahrenheit;
+  updateTempEmoji();
+  const button = document.getElementById('toggleTempUnit');
+  button.textContent = isFahrenheit ? '°C' : '°F';
+});
+
+applyCityArrow.addEventListener('click', () => {
+  updateCityName();
+});
```

Comment:
Not sure as to why this function, `updateCityName`, is wrapped in an anonymous function. You could just pass in the function reference like on `line 33`. 
```suggestion
applyCityArrow.addEventListener('click', updateCityName);
```